mkosi/config.py

```diff
@@ -3431,23 +3452,11 @@ def none_to_default(s: Optional[object]) -> str:
     return "default" if s is None else str(s)
 
 
-def line_join_list(array: Iterable[PathString]) -> str:
-    if not array:
-        return "none"
-
-    items = (str(none_to_none(cast(Path, item))) for item in array)
-    return "\n                                     ".join(items)
-
-
-def format_tree(tree: ConfigTree) -> str:
-    return f"{tree.source}:{tree.target}" if tree.target else f"{tree.source}"
-
-
-def line_join_tree_list(array: Sequence[ConfigTree]) -> str:
+def line_join_list(array: Iterable[object]) -> str:
     if not array:
         return "none"
 
-    items = [format_tree(tree) for tree in array]
+    items = (str(item) for item in array)
     return "\n                                     ".join(items)
```

Comment:
This wasn't too long before either, I think this can be safely inlined
```suggestion
    return "\n                                     ".join(str(item) for item in array)
```